docs/crio.8.md

```diff
@@ -116,8 +115,6 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--additional-devices**="": Devices to add to the containers  (default: [])
 
-**--allow-userns-annotation**: [EXPERIMENTAL] Allow usage of the annotation to setup user namespaces (default: false)
```

Comment:
`docs/crio.conf.5.md` also has to be updated, as it contains:
```
**allow_userns_annotation**=false
  [EXPERIMENTAL] Allow usage of the experimental feature for setting user namespaces through the io.kubernetes.cri-o.userns-mode annotation.  This is an experimental feature,
  do not enable in production.  It may be changed in the future without any notice.
```

Reply:
good catch, fixed!